src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php

```diff
@@ -49,32 +50,36 @@ public function getArguments(Request $request, callable $controller): array
 
         foreach ($this->argumentMetadataFactory->createArgumentMetadata($controller, ...$reflectors) as $metadata) {
             foreach ($this->argumentValueResolvers as $resolver) {
-                if (!$resolver->supports($request, $metadata)) {
+                if (!$resolver instanceof ValueResolverInterface && !$resolver instanceof TraceableValueResolver && !$resolver->supports($request, $metadata)) {
```

Comment:
the condition on `TraceableValueResolver` is useless, as this class is an instance of `ValueResolverInterface` and so already blocked by the first part.
But this might be a mistake, as it would mean that an ArgumentValueResolverInterface wrapped in a TraceableValueResolver would not call `supports` anymore.

Reply:
oops, updated